.pre-commit-config.yaml

```diff
@@ -79,9 +79,16 @@ repos:
         args: [--target-version, "5.0"]
 
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v1.9.0
+    rev: v1.10.0
     hooks:
       - id: mypy
+        args:
+          [
+            "--ignore-missing-imports",
```

Comment:
Mypy ran successfully before without this ignore, so why do we need this?

Also mypy is configured in pyproject.toml, I think we should keep the configuration there and not configure mypy in two different places.